Mert/fizz_buzz.rb

```diff
@@ -0,0 +1,30 @@
+def fizz_buzz(number)
+  if number.kind_of? Integer
+    if number>0
+      the_array = (1..number).to_a
+      p the_array
+
+      the_array.map! do |n|
+        if n%15==0
+          n = "FizzBuzz"
+        elsif n%3==0
+          n = "Fizz"
+        elsif n%5==0
+          n= "Buzz"
+        else
+          n=n
+        end
+      end
+      p the_array
+
+      return the_array
+    else return false
+    end
+  else return false
```

Comment:
Just a small suggestion: 
Instead of writing nested `if` statements like this
```ruby
if a == true
  if b == true
    # some commands
  end
end
```
it is much cleaner to write it like this:
```ruby
if a == false or b == false
  return;
end

# some commands
```